src/app/about/page.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from "next/image";
+import styles from "./About.module.css";
+
+export default function About() {
+  return (
+    <main className={styles.about}>
+      <h1 className={styles.about__title}>About</h1>
+      <div className={styles.about__description_container}> 
+        <p className={styles.about__description}>We are an organization that helps students learn and apply software engineering principles to real-world applications. We host weekly workshops on topics like software design to help bridge the gap between what students need to know for industry and what they're taught in classes. These concepts help with building complex software systems and better prepare members for team projects, internships, and careers in software development.
+
+</p>
+</div>
+     
+      <div className={styles.about__image_container}><Image src="group_photo.jpg" alt="A group of club members sitting in a room looking at the camera and posing" height={600} width={600} />
```

Comment:
[Suggestion]: We can probably get a bit better with the name/alt - I happen to know this photo is from the Fall 2022 planning meeting. Suggestion:

> fall_2022_planning_meeting.jpg
> SSD members (about 30) at our Fall 2022 planning meeting in CISE.

I think this better represents the meaning of the photo rather than just being a description, but open to suggestions. Mainly looking ahead to when we (hopefully) have a gallery of images and need to do a better job sorting through that.

Reply:
@WillBAnders The alt text has since been updated. However, the name of the file is "group_photo.jpg", not "fall_2022_planning_meeting.jpg". Will this discrepancy be an issue?

Reply:
No, I would just recommend the more explicit name to help if/when we add a gallery with multiple pictures where it becomes more important to distinguish each one. But the file can always be renamed at the time; it's more of a callout to prefer more detailed names when possible.